finish/pom.xml

```diff
@@ -141,11 +139,11 @@
             <plugin>
                 <groupId>net.wasdev.wlp.maven.plugins</groupId>
                 <artifactId>liberty-maven-plugin</artifactId>
-                <version>2.4</version>
+                <version>2.5</version>
                 <configuration>
                     <assemblyArtifact>
                         <groupId>io.openliberty</groupId>
-                        <artifactId>openliberty-runtime</artifactId>
+                        <artifactId>openliberty-kernel</artifactId>
```

Comment:
Are you sure about this change? Have you tested it with the guide?

Reply:
The support should be there.

Then, this doesn't mean that we will switch over.  We need to walk through the implications and make a call.

Reply:
Hi, I confirmed the change with Patrick Tiu, we've tested it with the kernel and it worked

Reply:
@evelinec  I have chatted with @ericglau who told me that you can use only the esa dependency to pull in other spec and api dependencies and keep using the openliberty-runtime but the esa and its transitive esa dependency will still be downloaded to your maven local cache, along with the full openliberty-runtime.

Reply:
For the record, you prefer not to use openliberty-kernel and installing only required features using ```install-feature``` plugin goal is because of the requirement to have the  ```<acceptLicense>true</acceptLicense>``` configuration paramenter.

We will take it as a feedback to the Liberty Maven plugin goal and considering to leave that requirement in the future version of Liberty Maven/Gradle plugin